Cargo.toml

```diff
@@ -1,7 +1,10 @@
 [package]
 name = "direct-storage"
-version = "0.6.0"
-authors = ["Tsukisoft GmbH <contact@tsukisoft.com>"]
+version = "0.6.0+DirectStorage-1.3.0"
+authors = [
+    "Tsukisoft GmbH <contact@tsukisoft.com>",
+    "Marijn Suijten <marijn@traverseresearch.nl>",
+]
```

Comment:
Since I've been actively helping maintain/update this crate, I hope you don't mind this. Thanks! :slightly_smiling_face: